services/visual-studio-app-center/visual-studio-app-center-releases-osversion.tester.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+// Note:
+// Unfortunately an Invalid user, invalid project, valid API token test is not possible due to the way Microsoft cache their responses.
+// For this reason 404 and 403 will instead both display 'project not found'
+
+t.create('[fixed] Example Release')
+  // This application will never have a new release created.
+  .get(
+    '/jct/test-fixed-android-react/8c9b519a0750095b9fea3d40b2645d8a0c24a2f3.json'
+  )
+  .expectBadge({
+    label: 'min version',
+    message: 'Android 4.1',
+  })
+
+t.create('[fixed] Example Release')
```

Comment:
Similar to my comment in the downloads tester, I think we should remove this test